### PR TITLE
Implemented Back Button for Student/Employee Detail Pages

### PR DESCRIPTION
**Pull request by tuckerstanley**
Added back button to employee/student detail Page. Had to change some of the CSS with the headers in order to get the formatting correctly. Couldn't figure out how to add the arrow icon while maintaining good spacing, let me know if you have any ideas about how to do that.

### CODE REVIEW

File: frontend/src/components/UserDetail/userDetail.module.css [hunk 1]
```diff
@@ -12,9 +12,29 @@
...
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 2.25rem 2.5rem;
+  font-weight: bold;
+}
+
+.header {
+  display: flex;
+  align-items: center;
```
Comment (mg2239) [inline frontend/src/components/UserDetail/userDetail.module.css:28]: to make sure the arrow and text are centered vertically

File: frontend/src/components/UserDetail/userDetail.module.css [hunk 2]
```diff
@@ -12,9 +12,29 @@
...
+  text-decoration: none;
+  color: inherit;
+  font-size: 1.75rem;
+  margin-bottom: 0;
+  margin-left: -28px;
```
Comment (mg2239) [inline frontend/src/components/UserDetail/userDetail.module.css:33]: this is how we can fit the arrow into the `.pageDivTitle` padding. the arrow is 24px wide + 4px right margin, so we add a negative margin so all the content in `.header` shifts left 28px, leaving the "Students" or "Employees" text in the same place

**Review by mg2239 — APPROVED** [review]
great work, im glad you were able to figure out most of the styling! i added the arrow styles myself just cause i thought it would be a bit complicated to explain without code, so i left comments on how it works below

File: frontend/src/components/UserDetail/userDetail.module.css (hunk 2 above)
Comment (tuckerstanley) [reply frontend/src/components/UserDetail/userDetail.module.css:33]: Sounds good, thanks for doing this.

**Pull request merged**